src/jenkins_job_insight/analyzer.py

```
@@ -91,11 +91,16 @@ async def _clone_into_subdir(ar: AdditionalRepo) -> None:
                 target,
                 depth=1,
                 branch=ar.ref,
+                token=ar.token or None,
             )
             cloned[ar.name] = target
             logger.info(f"Cloned additional repo '{ar.name}' into {target}")
         except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)